api/v1/openapi.yaml

```diff
@@ -783,7 +783,23 @@ paths:
             "$ref": "#/definitions/Error"
         '404':
           description: No DNS data with provided parameters found
-
+  "/fqdn/names":
+    get:
+      summary: List internal DNS selector representations
+      description: |
+        Retrieves the list of DNS-related fields (names to poll, selectors and
+        their corresponding regexes.
```

Comment:
Missing right parentheses

Reply:
👍 , having easy to understand names for end users is important. Will be fixed in next upload!

Reply:
Fixed!